### PR TITLE
hometask

### PR DESCRIPTION
**Pull request by MaxSkydan**

### CODE REVIEW

File: hometask3.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+"""Hometask format and f-strings."""
+import logging
+
+logging.basicConfig(level=logging.INFO, format='[%(asctime)s] %(levelname)s: %(message)s')
```
Comment (antohaUa) [inline hometask3.py]: logger was not yet examined by us - but ok if you want  - better to use it in such way
```
_log = logging.getLogger('Main')

console_handler = logging.StreamHandler()
log_format = logging.Formatter('[%(asctime)s %(levelname)s]: %(message)s')
console_handler.setFormatter(log_format)
_log.addHandler(console_handler)
_log.setLevel(logging.DEBUG)

text = 'text'
_log.info(text)
```

File: hometask3.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+# we have norway text in old style formatting
```
Comment (antohaUa) [inline hometask3.py]: Task description better to place into module docstring at the top.

File: hometask3.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+# re-write the same text as:
+# #1 string with format() call
+# #2 f-string
+# use linter(https://github.com/wemake-services/wemake-python-styleguide)
+# to check your new created python module for possible linter errors
+# try to run code from pycharm/command line
+
+norway_text = 'Automatisering akselererer %syeblikket da roboter vil erobre planeten v%sr. (%s)' % ('ø', 'å', 'Æ')
```
Comment (antohaUa) [inline hometask3.py]: Here and below linter gives us smth aka `E501 line too long (114 > 79 characters)` - you need to fix it

**Review by antohaUa — CHANGES_REQUESTED** [review]
In general good solution - just require cosmetic fixes mentioned by me in comments

**Comment by MaxSkydan** [comment]
fixed

File: hometask3.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
+"""Hometask format and f-strings.
+
+# we have norway text in old style formatting
+# re-write the same text as:
```
Comment (antohaUa) [inline hometask3.py]: comment characters better to remove - cause text inside docstring

**Review by antohaUa — APPROVED** [review]
Good

**Pull request merged**